### DISCUSSION TITLE
Element type in inp file

### DISCUSSION THREAD

**Discussion by inandi2** [neperfepx/neper#553] [Q&A] [answered]
Hi,
I want to generate an inp file using NEPER with only C3D10 type of element but what Neper is giving me is a combination of three different type of elements T3D3, CPE6, and C3D10. Is there any way I can solve this issue? Thank you.

**Comment by rquey** [comment, marked as answer]
What is your command?

The following example prints only C3D10 elements:
```
neper -T -n 2
neper -M n2-id1.tess -order 2 -for inp
```

**Comment by inandi2** [comment]
I have used the following command

```
neper -T -n 30 -id 1 -dim 3 -reg 1 -domain 'cylinder(3000,2500)' -morpho "size:normal(mean=0.008,sigma=0.002),columnar(z),gg"  -o 10grain -format tess,ori
neper -M 10grain.tess -elttype 'tri' -order 2 -rcl 0.50  -o 10grain -format 'inp','msh' -dim 'all'
```

**Reply by rquey** [reply]
`-dim all` is the culprit. Use `-dim 3` to print only the 3D elements (or simply don't use `-dim`, since 3 is the default...)
